pxblue/react-native-cli-templates
# Update to component/theme v6 syntax

Fixes #38 


#### Changes proposed in this Pull Request:
- Update to use the new icon syntax where appropriate\


#### Screenshots:


#### To Test:
- Follow the instructions in the Readme for testing template files locally
- I tested the auth typescript template locally and used that to propagate changes to the other templates, so I did not explicitly test the others. It would be good to just run through and make sure they all work.


**NOTE**: This PR depends on @pxblue/react-native-components@6.0.1 and @pxblue/react-native-auth-workflow@4.0.0 being published. We should update the packager versions in dependencies.json once they are available to replace the betas currently specified therein.
